services/GoogleSitemapService.cfc

```diff
@@ -145,22 +158,46 @@ component {
 		};
 	}
 
-	private function _addChildPages( required array haveAccessPages, required array childPages, string parentSearchEngineAccess, string parentAccessRestriction ) {
+	private struct function _getSitemapPriorityForPage( required string pageId ) {
```

Comment:
Any good reason this doesn't just return the string of the priority? Can see it used only in one place and it immediately gets the "sitemap_priority" key from the returned struct.